.github/actions/algolia-import-entity/action.yml

```diff
@@ -41,8 +41,18 @@ runs:
     - name: Import Entity
       shell: bash
       run: |
-        yarn export:entity:${{ inputs.app }} $ENTITY -f $ENTITY.json
-        jq -c '.[]' $ENTITY.json | algolia objects import $ALGOLIA_INDEX -F -
+        crn_entities=(event external-author interest-group news research-output team tutorial user working-group)
+        gp2_entities=(event external-user news output project user working-group)
+
+        [ "${{ inputs.app }}" == "crn" ] && entities=("${crn_entities[@]}") || entities=("${gp2_entities[@]}")
+
+        if [[ "$ENTITY" == "all" ]]; then
+          for entity in "${entities[@]}"; do yarn export:entity:${{ inputs.app }} $entity -f $entity.json
```

Comment:
you might want to rename `entity` to `iteratedEntity` or something like, I found it confusing that further down we have $ENTITY in caps